e2e/src/app.e2e-spec.ts

```diff
@@ -43,6 +45,8 @@ describe('Test App', () => {
 
             await browser.actions().mouseMove(element(by.cssContainingText('.value-component', /^1$/))).perform();
 
+            await browser.wait(EC.presenceOf(element(by.css('.action-bubble'))), timeout, 'Wait for action bubble to be visible.');
+
             expect(element(by.css('.action-bubble')).isDisplayed()).toBeTruthy();
```

Comment:
Sometimes the tests failed here. Since this only happens from time to time, I have the suspicion it cold be a timing issue when trying to find `.action-bubble`.

Reply:
interesting, let see if it fixes the issue :)

Reply:
hard to say since this seems to be intermittent

Reply:
The strange thing is that `element(by.css('.action-bubble')).isDisplayed()` returned `false` and the assertions was not met. So I am not sure whether the element was not found at all or was found but not displayed.

`presenceOf` makes sure that the element can be found within the timeout, but if the element takes some time to appear, it won't fix that issue.